### ISSUE TITLE
Regex conversion loses plus character

### ISSUE THREAD

**Issue by johanfforsberg** [brunato/xmlschema#57]
I ran in to the following issue when validating a schema which checks a date string with a pattern like this: 
`<xs:pattern value=".+T.+(Z|[+-].+)"/>`
I get this kind of error:
```
failed validating '2015-12-31T13:32:26+02:00' with XsdPatternsFacet(['.+T.+(Z|[+-].+)']).

Reason: value don't match any pattern of ['.+T.+(Z|[+-].+)'].
```
This pattern seems fine and matches the example if I try it with the python `re` module directly. 

As far as I can tell, this happens in `regex.get_python_regex` which produces a new regex like this: `'^[^\\r\\n]+T[^\\r\\n]+(Z|[\\-][^\\r\\n]+)$'` which appears to be missing the "+" character within the brackets.

**Comment by brunato** [comment]
Caught this error. It was at line 74 of *codepoints.py* module:

https://github.com/brunato/xmlschema/blob/f2c5d19b677f33745e758d5c8678059b92850164/xmlschema/codepoints.py#L74

The first test has to be changed to `last_index <= 1`. I'll include this in the next version, adding also a test with your case.

Thanks

**Comment by brunato** [comment]
The fix is now available on release 0.9.26.
